package.json

```diff
@@ -48,5 +48,13 @@
 		"typescript": "^5.5.4",
 		"vite": "^4.3.9"
 	},
+	"resolutions": {
+		"micromatch": "4.0.8",
+		"ws": "8.17.1"
+	  },
+	"packageComments":{
+		"micromatch": "Overriding the version of micromatch to 4.0.8 to fix a vulnerability issue, Remove the override once the parent packages @changesets/cli and @typescript-eslint/parser are updated to a version that use latest micromatch.",
+		"ws": "Overriding the version of ws to 8.17.1 to fix a vulnerability issue. Remove the override once the parent package tailwind is updated to a version that use latest ws."
```

Comment:
I think we should just make an issue/PR on there side to update these 🙏 
This way we don't add this Tech Debt to our repo (which already has quite a lot ;(

Reply:
https://github.com/changesets/changesets/issues/1448